### DISCUSSION TITLE
What's your favorite dinosaur

### DISCUSSION THREAD

**Discussion by Quinadia** [Illinois-State-Solar-Car/Steamer-Trunk#19] [Q&A] [answered]
Howdy fellas (@Illinois-State-Solar-Car/mechanical )
to get to know the things about you that really matter tell us your favorite dinosaur and why.

**Comment by bwetz23** [comment, marked as answer]
easy, brontosaurus. only because he is a leafy guy

**Comment by kebarr3** [comment]
The GOAT, I think we all know which one I’m talking about. The Halszkaraptor
